code/datums/helper_datums/construction_datum.dm

```diff
@@ -375,11 +375,6 @@
 	else
 		var/list/spawn_step
 		var/new_index = (diff == FORWARD ? index - 1 : index + 1)
-		if(new_index == 0)
-			message_admins("Holy shit [src]/([src.type]) is trying to set its new index to 0! how the fuck did this happen? I don't know, our direction is [diff==FORWARD?"forward":"backward"] old index was [index]. User is [formatPlayerPanel(user,user.ckey)], itemused [used_atom], step [given_step]")
-			spawn_result(user)
```

Comment:
By removing this does it still spawn the completed item later on?

Reply:
Yeah. It does it in [update_index()](https://github.com/vgstation-coders/vgstation13/blob/7d3124ce91f6fc3bacadfcbcbacb057f30d638b4/code/datums/helper_datums/construction_datum.dm#L214)
```
if(index==0)
	spawn_result(user)
```